fullstack_app/client/src/Components/App.tsx

```diff
@@ -17,45 +17,40 @@ const App: FC = () => {
 
   useEffect(() => {
     async function initList() {
-      let defaultList = await fetchFruit()
-      setFruitList(defaultList.data)
+      let res = await fetchFruit()
```

Comment:
As we're not reassigning `res` during this block of code, we can use `const` instead of let

```suggestion
      const res = await fetchFruit()
```

In fact a good way of working with ES5+ Javascript is to always use const until const doesn't work, then try switching to let. You'll find 90% of your variables will probably work fine as a constant

Reply:
Thanks! Have made that change and a couple of similar ones